recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.6,<3.10
```

Comment:
No, remove here the version range completely. It should just stay in run section.  Instead add a skip for python <36